bin/nexus-repositories-create

```diff
@@ -43,28 +43,28 @@ function find_repo(){
     # If changes are needed, ensure that testing is done on both Nexus 3.25.0 (csm-1.2) and
     # 3.38.0 (csm-1.3) since this is used in both versions.
     if [[ $# -ne 3 ]]; then
-        echo -n >&2 "ERROR Missing arguments. Expecting repo_format repo_type repo_name. Arguments passed were: $@"
+        echo -n >&2 "Missing arguments. Expecting repo_format repo_type repo_name. Arguments passed were: $@"
         return 1
     fi
-    if [[ -z "$1" ]]; then echo -n >&2 "ERROR Empty repo_format argument"; return 1; fi
-    if [[ -z "$2" ]]; then echo -n >&2 "ERROR Empty repo_type argument"; return 1; fi
-    if [[ -z "$3" ]]; then echo -n >&2 "ERROR Empty repo_name argument"; return 1; fi
+    if [[ -z "$1" ]]; then echo -n >&2 "Empty repo_format argument."; return 1; fi
+    if [[ -z "$2" ]]; then echo -n >&2 "Empty repo_type argument."; return 1; fi
+    if [[ -z "$3" ]]; then echo -n >&2 "Empty repo_name argument."; return 1; fi
     repo_format=$1
     repo_type=$2
     repo_name=$3
 
-    echo -n >&2 "DEBUG Looking for ${repo_format}/${repo_type} repository: ${repo_name} ..."
+    echo -n >&2 "Looking for ${repo_format}/${repo_type} repository: ${repo_name}..."
```

Comment:
IUF will not be able to recognize this message if DEBUG is removed. End space removal is fine.

Reply:
As @rajeshranganathan-hpe indicates, all messages should start with INFO, ERROR, WARNING or DEBUG so the messages are properly sent to the correct places.

Reply:
I think there's some confusion here. Please note, most `echo` commands in these files are provided with `-n` CLI option, which means "Do not print the trailing newline character". I.e. if there are 2 consecutive `echo -n` commands, output from 2nd echo continues at the same line. If message starts from ERROR|WARNING|DEBUG, this will be in the middle of printed line, not at the beginning. This is exactly what makes output of this script so hard to read.